### PR TITLE
Email templates done and error fixed in Nepal locales added

### PR DESCRIPTION
**Pull request by AshishMhrzn10**
I did the job of email templates and fixed previous error and pushed it to 0.7.x branch

### CODE REVIEW

**Review by spielers — CHANGES_REQUESTED** [review]
The translation is Good but you should have to remove the extra whitespace

**Comment by AshishMhrzn10** [comment]
sorry, I didn't get it. Can u point out some ?I will fix it asap. Thanks in advance.

File: app/config/locale/translations/templates/np.email.auth.invitation.tpl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<p>
+    नमस्कार {{name}},
+</p>
+<p>
+    यो मेल तपाईंलाई पठाइएको थियो किनभने <b>{{owner}}</b>ले  तपाईंलाई <b>{{team}}</b> टीममा रहेको {{project}} प्रोजेक्टमा टीमको सदस्य बन्न आमन्त्रित गर्न चाहन्छ।
```
Comment (spielers) [inline app/config/locale/translations/templates/np.email.auth.invitation.tpl:5]: remove extra space between ले  तपाईंलाई => ले तपाईंलाई

File: app/config/locale/translations/templates/np.email.auth.recovery.tpl [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<p>
+    नमस्कार {{name}},
+</p>
+<p>
+   तपाईंको {{project}}को  पासवर्ड रिसेट गर्नका लागि यो लिंक अनुसरण गर्नुहोस्।
```
Comment (spielers) [inline app/config/locale/translations/templates/np.email.auth.recovery.tpl:5]: remove extra space from को  पासवर्ड =>को पासवर्ड

**Review by spielers — CHANGES_REQUESTED** [review]

File: app/config/locale/translations/templates/np.email.auth.recovery.tpl (hunk 2 above)
Comment (AshishMhrzn10) [reply app/config/locale/translations/templates/np.email.auth.recovery.tpl:5]: ohh thanks i will do that.

Comment (AshishMhrzn10) [reply app/config/locale/translations/templates/np.email.auth.recovery.tpl]: @spielers please check now. if any error then i would happily resolve again. Thanks!

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Comment by AshishMhrzn10** [comment]
@eldadfux , Thank you so much sir.

**Pull request merged**